src/main.ts

```diff
@@ -222,7 +214,7 @@ function main(args: string[]) {
   }
 
   if (settings.externsPath) {
-    fs.writeFileSync(settings.externsPath, jsFiles[internalExternsFileName]);
+    fs.writeFileSync(settings.externsPath, externs);
```

Comment:
Since this is now conditional, it's worth highlighting that we may silently miscompile code if you forget to include externs.  I wonder if it's worth requiring the --externsPath flag.